.travis.yml

```diff
@@ -16,9 +16,11 @@ matrix:
   include:
     - php: 5.5
       env:
+        - SERVICE_MANAGER_VERSION="^2.7.3"
```

Comment:
All references to 2.7.3 and 3.0 need to be updated to 2.7.5 and 3.0.3, respectively, as those are the versions adding the `CommonPluginManagerTrait`. I can take care of that on merge, however.